docs/source/checks/vision/data_integrity/plot_property_label_correlation.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-.. _plot_vision_feature_label_correlation:
+.. _plot_vision_property_label_correlation:
```

Comment:
Shouldn't this be _vision_property... ? Can you compare to other vision checks in the new ver?

Reply:
Fixed